src/libsyntax/diagnostic_list.rs

```diff
@@ -93,6 +93,38 @@ For more information about the cfg attribute, read:
 https://doc.rust-lang.org/reference.html#conditional-compilation
 "##,
 
+E0541: r##"
+An unknown meta item was used.
+
+Erroneous code example:
+
+```compile_fail,E0541
+#[deprecated(
+    since="1.0.0",
+    // error: unknown meta item
+    reason="Example invalid meta item. Should be 'note'")
+]
+fn deprecated_function() {}
+```
+
+Meta items are the key/value pairs inside of an attribute. The keys provided
```

Comment:
Why not "key-value pairs" instead of "key/value pairs".

Reply:
It's what was used in the reference. Should probably change it there as well.